packages/netsuite-adapter/src/config.ts

```diff
@@ -258,23 +328,23 @@ const updateConfigFromFailures = (
     }, values.isDefined)
   }
 
-  const currentSkipList = configToUpdate.value[SKIP_LIST]
-  const newSkipList: Partial<NetsuiteQueryParameters> = currentSkipList !== undefined
-    ? _.cloneDeep(currentSkipList)
-    : {}
+  const currentFetch = configToUpdate.value[FETCH]
+  const currentExclude = currentFetch?.[EXCLUDE] !== undefined
+    ? _.cloneDeep(currentFetch?.[EXCLUDE])
+    : { types: [], fileCabinet: [] }
 
-  const suggestedSkipList: NetsuiteQueryParameters = suggestions.skipList
-  if (suggestedSkipList.filePaths.length > 0) {
-    newSkipList.filePaths = [
-      ...makeArray(newSkipList.filePaths),
-      ...suggestedSkipList.filePaths,
-    ]
-  }
 
-  if (!_.isEmpty(suggestedSkipList.types)) {
-    newSkipList.types = mergeTypeToInstances(newSkipList.types ?? {}, suggestedSkipList.types)
+  const suggestedExclude = suggestions[FETCH]?.[EXCLUDE]
+  const newExclude = combineQueryParams(currentExclude, suggestedExclude)
+
+  if (currentFetch === undefined) {
+    configToUpdate.value[FETCH] = {
+      [INCLUDE]: fetchDefault[INCLUDE],
+      [EXCLUDE]: newExclude,
+    }
```

Comment:
Shouldn't this be:

```suggestion
    configToUpdate.value[FETCH] = {
      [EXCLUDE]: newExclude,
    }
```
?